client/src/components/StandardFAQ/StandardFAQ.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+import { SomeThingsToKeepInMind } from "src/panels/panel_declarations/common_panel_components";
```

Comment:
Since this is moving from an infographic-specific thing to a site-wide thing, `SomeThingsToKeepInMind` should be removed from `common_panel_components`. That's essentially what this component should be directly.

In practice, `SomeThingsToKeepInMind` in the infographics always contains a `KeyConceptList` so the existing use cases of it there should be directly replaced with this FAQ component.